checker/src/main/java/org/checkerframework/checker/initialization/InitializationAnnotatedTypeFactory.java

```diff
@@ -110,11 +114,12 @@
     /**
      * String representation of all initialization annotations.
      *
-     * <p>{@link UnknownInitialization} {@link UnderInitialization} {@link Initialized} {@link
-     * FBCBottom}
+     * <p>{@link UnknownInitialization}, {@link UnderInitialization}, {@link Initialized}, {@link
+     * FBCBottom}, {@link PolyInitialized}.
      *
-     * <p>This is used to quickly check of an AnnotationMirror is one of the initialization
-     * annotations without having to repeatedly convert them to strings.
+     * <p>This is used to quickly check if an AnnotationMirror is one of the initialization
+     * annotations without having to repeatedly convert them to strings. This field is initialized
```

Comment:
Undo the new sentence here, instead add the `@CanonicalName` and add that fact to the first sentence of this javadoc.

Reply:
Thanks for the suggestion. Done